com.cuperino.qprompt.json

```diff
@@ -0,0 +1,40 @@
+{
+   "command": "qprompt",
+   "finish-args": [
+      "--share=ipc",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--filesystem=host",
+      "--share=network",
+      "--device=all"
+   ],
+   "id": "com.cuperino.qprompt",
+   "cleanup" : [
+      "/include",
+      "/lib/pkgconfig",
+      "/man",
+      "/share/doc",
+      "/share/gtk-doc",
+      "/share/man",
+      "/share/pkgconfig",
+      "*.la",
+      "*.a"
+   ],
+   "modules": [
+      {
+         "buildsystem": "cmake-ninja",
+         "name": "qprompt",
+         "sources": [
+            {
+               "type": "git",
+               "url": "https://github.com/Cuperino/QPrompt.git",
+               "tag": "v1.0.0",
+               "commit": "c886f9b692dc8900af69f01b6af26f19f220090f"
+            }
+         ]
+      }
+   ],
+   "runtime": "org.kde.Platform",
+   "runtime-version": "5.15",
```

Comment:
this whol section is usually at the top of the file

Reply:
Moved section to the top, right below id.